### PR TITLE
Show actual error response when trying

### PR DESCRIPTION
**Pull request by furkanayhan**
before:
screen shot 2017-05-12 at 10 58 36

after:
screen shot 2017-05-12 at 11 34 50

### CODE REVIEW

**Comment by webron** [comment]
@furkanayhan Thank you for taking the time to submit the PR. To help review it, do you have a live project we can look at that reproduces the issue? Or otherwise point to how to reproduce it with the petstore?

In order to avoid future breaking changes too, it would be great if you can add a test as well.

**Comment by furkanayhan** [comment]
hi @webron 
actually you can reproduce the issue with petstore (http://petstore.swagger.io/)

Try with this request:
screen shot 2017-05-13 at 19 12 02

You'll get the response code, but not the response body.
screen shot 2017-05-13 at 19 11 02

**Comment by webron** [comment]
@furkanayhan Thanks, I should have updated that I was able to reproduce it, but it looks like something is a bit off. Can't really figure out why the 'Error: Unsupported Media Type' message is there.

File: src/core/components/live-response.jsx [hunk 1]
```diff
@@ -37,6 +38,7 @@ export default class LiveResponse extends React.Component {
     const returnObject = headersKeys.map(key => {
       return <span className="headerline" key={key}> {key}: {headers[key]} </span>
     })
+    const hasHeaders = returnObject.length !== 0
```
Comment (shockey) [inline src/core/components/live-response.jsx]: Can we make this `returnObject > 0`? I'm concerned about a possible edge case where `returnObject` is null or undefined.

File: src/core/components/live-response.jsx [hunk 2]
```diff
@@ -67,15 +69,15 @@ export default class LiveResponse extends React.Component {
                   </span>
                 }
                 {
-                  !body || isError ? null
```
Comment (shockey) [inline src/core/components/live-response.jsx]: I'd like to see the ternary operators here be flipped around - the logical not in the comparison argument makes them hard to read.

File: src/core/components/live-response.jsx [hunk 3]
```diff
@@ -67,15 +69,15 @@ export default class LiveResponse extends React.Component {
...
+                  !body ? null
```
Comment (shockey) [inline src/core/components/live-response.jsx]: I'd like to see the ternary operators here be flipped around - the logical not in the comparison argument makes them hard to read.

File: src/core/components/live-response.jsx [hunk 4]
```diff
@@ -67,15 +69,15 @@ export default class LiveResponse extends React.Component {
...
                         : <ResponseBody content={ body }
                                         contentType={ contentType }
                                         url={ url }
                                         headers={ headers }
                                         getComponent={ getComponent }/>
                 }
                 {
-                  !headers ? null : <Headers headers={ returnObject }/>
+                  !hasHeaders ? null : <Headers headers={ returnObject }/>
```
Comment (shockey) [inline src/core/components/live-response.jsx]: I'd like to see the ternary operators here be flipped around - the logical not in the comparison argument makes them hard to read.

File: src/core/components/live-response.jsx (hunk 2 above)
Comment (shockey) [inline src/core/components/live-response.jsx]: I'd like to see the ternary operators here be flipped around - the logical not in the comparison argument makes them hard to read.

**Comment by webron** [comment]
@furkanayhan do you have the time to look into the questions here?

**Comment by shockey** [comment]
CC: #3041

**Comment by furkanayhan** [comment]
@webron @shockey I've made changes and updated branch

**Comment by plassa-b** [comment]
Any news on this PR ? It looks good

**Comment by shockey** [comment]
Thanks for the improvement, @furkanayhan! 

This code will go live when we release a new version of Swagger-UI on Friday.

**Pull request merged**